test/functional/cmdLineTests/build.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 
 <!--
-  Copyright (c) 2016, 2018 IBM Corp. and others
+  Copyright (c) 2016, 2018, 2019 IBM Corp. and others
```

Comment:
change to 
Copyright (c) 2016, 2019 IBM Corp. and others

Reply:
Have fixed thanks